README.md

```diff
@@ -85,11 +92,11 @@ yarn
 cd ios && pod install
 ```
 
-4. It is required to have `WebRTC.framework` M112 builds. You can find it in `libWebRTC.zip` inside the `ios` folder.
+1. It is required to have `WebRTC.framework` M112 builds. You can find it in `ios/libWebRTC.zip`.
 
-5. To download the zip file from Git LFS is needed to run the command `git lfs pull`.
+2. To download the zip file from Git LFS is needed to run the command `git lfs pull`.
 
-3. Unzip libWebRTC inside the `ios` folder from this project.
+3. Unzip `libWebRTC` inside the `ios` folder from this project.
 #### iOS
 
 1. Open Xcode.
```

Comment:
It should be mentioned for adding WebRTC as a framework step to check which Mac architecture is being used.

Reply:
Good idea. I added a link to the remark (**).